base/src/com/google/idea/blaze/base/model/primitives/Label.java

```diff
@@ -107,7 +107,7 @@ public String externalWorkspaceName() {
     }
     int slashesIndex = label.indexOf("//");
     logger.assertTrue(slashesIndex >= 0);
-    return label.substring(1, slashesIndex);
+    return label.substring(0, slashesIndex).replaceFirst("^@+", "");
```

Comment:
I believe there's something wrong here, this line probably should not be changed since in bazel external workspace name should has only one `@` symbol

Reply:
I see that it's the way bazel 7 started to report external workspace targets. Investigating.

Reply:
So my conclusion here is the following:

1. Please drop this change from this patchset.
2. Create new issue describing issue about bazel 7 -- this code works fine with bazel 6. 
3. Please check any other occurrences of `@` strip and adjust them. 
4. Should we introduce some special function that expects exactly 1 or exactly 2 `@` signs?